src/core/batch/parquetSchema.ts

```diff
@@ -127,20 +151,18 @@ export const ReactionBloomFilterOptions = {
  * @returns The corresponding parquet schema
  */
 export const getSchemaFor = (announcementType: AnnouncementType): Schema => {
-  switch (announcementType) {
-    case AnnouncementType.GraphChange:
-      return GraphChangeSchema;
-    case AnnouncementType.Broadcast:
-      return BroadcastSchema;
-    case AnnouncementType.Reply:
-      return ReplySchema;
-    case AnnouncementType.Reaction:
-      return ReactionSchema;
-    case AnnouncementType.Profile:
-      return ProfileSchema;
-  }
-
-  throw new InvalidAnnouncementTypeError(announcementType);
+  const schemas: Record<AnnouncementType, Schema> = {
+    [AnnouncementType.Tombstone]: TombstoneSchema,
+    [AnnouncementType.GraphChange]: GraphChangeSchema,
+    [AnnouncementType.Broadcast]: BroadcastSchema,
+    [AnnouncementType.Reply]: ReplySchema,
+    [AnnouncementType.Reaction]: ReactionSchema,
+    [AnnouncementType.Profile]: ProfileSchema,
+  };
+
+  if (schemas[announcementType] === undefined) throw new InvalidAnnouncementTypeError(announcementType);
+
+  return schemas[announcementType];
```

Comment:
Shorter, but harder to read perhaps
```suggestion
  return schemas[announcementType] || throw new InvalidAnnouncementTypeError(announcementType);
```

Reply:
The parser doesn't seem to like it:

```
/Users/rosalinekarr/Development/Liberty/sdk-ts/src/core/batch/parquetSchema.ts
  163:38  error  Parsing error: Expression expected
```